docs/index.md

```diff
@@ -117,7 +117,13 @@ For more information about the project and available options refer to this [repo
         aspect 'software.amazon.lambda:powertools-logging:{{ powertools.version }}'
         aspect 'software.amazon.lambda:powertools-tracing:{{ powertools.version }}'
         aspect 'software.amazon.lambda:powertools-metrics:{{ powertools.version }}'
+        implementation 'software.amazon.lambda:powertools-logging:{{ powertools.version }}'
```

Comment:
is implementation needed ?

Reply:
maybe not, i saw it was in the cookiecutter examples. it works without it.

Reply:
Hmm i thought i removed it from all the usage in cookiecutter. I will check and fix that if its still present.

Reply:
I checked, its not present in cookiecutter project as well.

Reply:
cool. i will remove it then :)